plugins/metastore-locator/core/src/main/java/org/pentaho/metastore/locator/api/impl/MetastoreLocatorImpl.java

```diff
@@ -70,10 +70,11 @@ public IMetaStore getMetastore() {
 
   @Override
   public IMetaStore getMetastore( String providerKey ) {
-    IMetaStore metaStore = getExplicitMetastore( MetastoreLocator.REPOSITORY_PROVIDER_KEY );
+    //Look for VFS metastore.
+    IMetaStore metaStore = getExplicitMetastore( MetastoreLocator.VFS_PROVIDER_KEY );
+    System.err.println( "This is running the current code totally dude" );
```

Comment:
probably don't want to check this in. :)

Reply:
Yep oops haha